app/src/app/navtitle/navtitle.component.sass

```diff
@@ -3,6 +3,8 @@
 @import '../../sizes'
 @import '../../typo'
 
+$breakpoint-tablet: 500px
+
```

Comment:
Such a variable will be needed for other components. We don't want each component to have its own breakpoints, because that would result in a flickering application. We want to have just a few breakpoints. Therefore, we must keep them in a shared file, similar to sizes.sass and typo.scss that are imported here. I propose to name this new file responsive.sass.